android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt

```diff
@@ -80,6 +81,11 @@ class PlayerModule(context: ReactApplicationContext) : BitmovinBaseModule(contex
         val defaultMetadata = analyticsConfigJson?.getMap("defaultMetadata")?.toAnalyticsDefaultMetadata()
         isMediaSessionPlaybackEnabled = playerConfigJson?.getMap("lockScreenControlConfig")
             ?.toLockScreenControlConfig()?.isEnabled ?: false
+        isBackgroundPlaybackEnabled = if (isMediaSessionPlaybackEnabled) {
+            isMediaSessionPlaybackEnabled
+        } else {
+            playerConfigJson?.getMap("playbackConfig")?.getBoolean("isBackgroundPlaybackEnabled") ?: false
+        }
```

Comment:
### Open Question about Config Design

Currently, I am making the media session flag forcibly treat the background flag as true when enabled, but that still has to be decided.

This way makes sense for me since media-session is a form of background-playback.